config.go

```diff
@@ -457,6 +457,7 @@ func NewConfig() *Config {
 	c.Consumer.Return.Errors = false
 	c.Consumer.Offsets.AutoCommit.Enable = true
 	c.Consumer.Offsets.AutoCommit.Interval = 1 * time.Second
+	c.Consumer.Offsets.CommitInterval = 1 * time.Second
```

Comment:
@reenjii I'm wondering if we need to add back the initial value....
We added this here: https://github.com/Shopify/sarama/pull/1590
![image](https://user-images.githubusercontent.com/950087/74743470-726de780-522e-11ea-90ce-880c6bcdee1e.png)

But if we add back the initial value that would means that by default sarama will *always* log a deprecated message.

Couldn't this be solved by changing the value when creating the sarama-cluster config? as the field was added back

Reply:
Yeah I understand that it would look wierd.
I'll add this default value in our configuration then.